src/oc/web/components/user_profile.cljs

```diff
@@ -194,7 +194,7 @@
                   {:type "text"
                    :tab-index 1
                    :on-change #(change! s :first-name (.. % -target -value))
-                   :value (:first-name current-user-data)}]]]
+                   :value (or (:first-name current-user-data) "")}]]]
```

Comment:
This cleanup should be done once in the user store.  Before changing the `:current-user-data` check to see if the `:first-name` is nil and add the empty string.  Then change the app state. You only do it once instead of all the times you change it here.